lib/ansible/modules/lineinfile.py

```diff
@@ -44,6 +44,17 @@
     type: str
     aliases: [ regex ]
     version_added: '1.7'
+  search_string:
+    description:
+      - The literal string to look for in every line of the file. This does not have to match the entire line.
+      - For C(state=present), the line to replace if the string is found in the file. Only the last line found will be replaced.
+      - For C(state=absent), the line(s) to remove if the string is in the line.
+      - If the literal expression is not matched, the line will be
+        added to the file in keeping with C(insertbefore) or C(insertafter)
+        settings.
+      - Mutually exclusive with C(backrefs) and C(regexp).
+    type: str
+    version_added: '2.11'
```

Comment:
Hmm, this is also contained in `devel`. Maybe you need to rebase to clean up the dif.

Reply:
Yeah, I'm not really sure why those were showing up as changes because my branch has it and so does `devel`.

Reply:
Followed the rebasing instructions [here](https://docs.ansible.com/ansible/devel/dev_guide/developing_rebasing.html). Hopefully that should fix the inconsistencies we were seeing